magento/pwa-studio
# Merge Carts on log in

## Description
Cart Merge feature

## Related Issue





Closes #2190 

## Acceptance 



### Verification Stakeholders

@sirugh 
@tjwiebell 
### Specification




### Verification Steps


1. Login and add products to cart
2. logout and add products as guest
3. Login again
4. cart should be merged with both list of products

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
